src/shogun/distance/AttenuatedEuclideanDistance.h

```diff
@@ -17,7 +17,9 @@
 #include <shogun/distance/RealDistance.h>
 #include <shogun/features/DenseFeatures.h>
 
-namespace shogun
+using namespace shogun;
```

Comment:
@lisitsyn Is use of the scope resolution operator <code>::</code> for specifying type of the fields of the different classes okay or is it something that you discourage?